src/vstest.console/TestPlatformHelpers/TestRequestManager.cs

```diff
@@ -247,8 +247,10 @@ public bool RunTests(TestRunRequestPayload testRunRequestPayload, ITestHostLaunc
 
             this.CollectMetrics(requestData, runConfiguration);
 
-            if (!commandLineOptions.IsDesignMode && string.IsNullOrWhiteSpace(runsettings))
+            if (!commandLineOptions.IsDesignMode)
```

Comment:
Do we have any test covering this scenario?

Reply:
No we can't test fakes in PR/CI.